### PR TITLE
iFlight H7 Thunder

### PR DESCRIPTION
**Pull request by andyp1per**
Support for the iFlight Thunder 2RAW H743

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md:10]: it has no osd chip

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md:11]: why not? shouldnt we enable it (even requiring creating a driver before merging) since its the only logging means?

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md:51]: no its does not

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md:69]: add vtx pwr switch section

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat:53]: I would not merge this until its understood and corrected

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat:55]: this needs to be set to DisplayPort by default

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat:137]: remove

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat:138]: remove

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat:139]: remove

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat:120]: remove all this....does not exist

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md]: Good catch! Fixed

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md:10]: No driver. Vendor wants to provide the board without logging support

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md]: Fixed

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat]: Fixed. It was the non-existent OSD that was causing the issue

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat]: Fixed

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat]: Fixed

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat]: Fixed

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat]: Fixed

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat]: Fixed

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md:10]: Will be boldly noted in the wiki

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md:68]: Do you have an example @Hwurzburg ?

Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md:68]: https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405WING/Readme.md

Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md:68]: Thx, fixed

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md:88]: should mention RELAY2 is pre-setup for this

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md:88]: Fixed

**Review by Hwurzburg — APPROVED** [review]

**Pull request merged**
